number_trivia_app/linux/flutter/generated_plugin_registrant.cc

```diff
@@ -0,0 +1,11 @@
+//
```

Comment:
Seems you re-added unnecessary files. Please remove files from linux, macos and windows